### ISSUE TITLE
Change metabox text color to black to pass accessibility

### ISSUE THREAD

**Issue by nicolepaschen** [BoldGrid/boldgrid-theme-framework#755]
We use a medium gray color for text in the page Attributes and Page Header metaboxes. Please update to black to pass accessibility contrast. 
![change-to-black](https://user-images.githubusercontent.com/20601160/195172584-8adf5e72-b3a5-49fa-b3b3-e5a4151e3ff2.PNG)
